README.rst

```diff
@@ -52,6 +52,8 @@ To use edx-sphinx-theme for a repository's documentation:
 * Use the ``AUTHOR`` constant where appropriate in conf.py
   (this default is only provided as a convenience; the repository is free
   to use another value if appropriate).
+* To add an "Edit on Github" link to every page, add a dict called ``html_context``,
+  as detailed in the following example.
```

Comment:
RST should wrap itself. Was this a quality complaint?

Reply:
Yes. I got a max line check error.

Reply:
doc8.checks.CheckMaxLineLength = 1

Reply:
Is this still a valid config: https://github.com/edx/edx-sphinx-theme/blob/master/tox.ini#L5

Reply:
This is from cookiecutter: https://github.com/edx/edx-cookiecutters/blob/b7a774ecf492c66496eb38b662c746f4f4589cf1/python-template/%7B%7Bcookiecutter.placeholder_repo_name%7D%7D/tox.ini#L4-L6